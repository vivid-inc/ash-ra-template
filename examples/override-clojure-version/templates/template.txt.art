(clojure-version) returns <(= (clojure-version) )>
