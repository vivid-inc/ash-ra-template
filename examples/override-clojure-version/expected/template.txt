(clojure-version) returns 1.10.1
